packages/react/src/components/Masthead/HeaderNavContainer.js

```diff
@@ -30,75 +30,140 @@ const HeaderNavContainer = ({ children }) => {
   const [io, setIO] = useState(null);
   const [position, setPosition] = useState(0);
   const buttonSize = 48; // 40px(width) + 8px(gradient)
+  const pageIsRTL = document.dir === 'rtl';
```

Comment:
@annawen1 this is causing errors in NextJS

![image](https://user-images.githubusercontent.com/1641214/128535822-908b2b9b-9d25-4bcd-85b2-dac759a07fe2.png)